docs/manual/manual_dronecan.rst

```diff
@@ -44,11 +44,274 @@ Servo modules do not support DroneCAN, as shown in the table below.
+
+The values represent the speed and direction to command the motor to, normalized over a range of [-8192, 8191], with -8192 being full speed in reverse and 8191 being full speed forward.
+
+Refer to the `uavcan.equipment.esc.RawCommand section of Standard Data Types <https://dronecan.github.io/Specification/7._List_of_standard_data_types/#rawcommand>`_ in the specification 
+for more details
+
+Service Requests
+*****************
+Service requests are messages sent to a specific target node from another node, and to which the sending node expects to receive a response message.
+
+uavcan.protocol.GetNodeInfo (Data Type ID = 1)
+################################################
+This request has no payload fields. The response message from the receiving node should include the status of the node as defined by the NodeStatus message, the 
+software and hardware version of the node, and the name of the node. For Vertiq modules, the name of each node is currently “iq_motion.esc”.
+
+Refer to the `uavcan.protocol.GetNodeInfo section of Standard Data Types <https://dronecan.github.io/Specification/7._List_of_standard_data_types/#getnodeinfo>`_ in the specification 
+for more details
+
+uavcan.protocol.param.GetSet (Data Type ID = 11)
+##################################################
+Used to get or set the value of a configuration parameter using either the index or the name of the parameter. The configuration parameters currently available on standard 
+Vertiq modules are listed in the :ref:`dronecan_configuration_parameters` section below. The request message should contain the index of the parameter or the name of the parameter depending 
+on how you are accessing it, and it should contain a value to set the parameter to if you are setting it. **Parameter indices are not guaranteed to remain consistent across 
+firmware versions. Indices should only be used for parameter discovery, when accessing the parameter directly it is recommended to always use the name**
+
+The response message will contain the value of the parameter, information about its default, minimum, and maximum values, and the name of the parameter.
+
+Refer to the `uavcan.protocol.param.GetSet section of Standard Data Types <https://dronecan.github.io/Specification/7._List_of_standard_data_types/#getset>`_ in the specification 
+for more details
+
+uavcan.protocol.RestartNode (Data Type ID = 5)
+###############################################
+This request has one field, its “magic number”. This field is an unsigned 40 bit integer. The magic number is used to identify that this is an intentional restart request, 
+and we have also extended it to allow for two different types of reboots. One magic number performs a normal reboot that simply restarts into the normal application firmware, 
+and the other reboots the device into the ST bootloader, which allows it to be programmed with new firmware. The reboot-into-bootloader can be useful for firmware updates. 

```

Comment:
link to the ST bootloader information page? This is sort of meaningless for anyone who's never seen or worked with ST before.